controlplane/kubeadm/api/v1beta1/kubeadm_control_plane_types.go

```diff
@@ -91,6 +110,10 @@ type KubeadmControlPlaneSpec struct {
 	// +optional
 	// +kubebuilder:default={type: "RollingUpdate", rollingUpdate: {maxSurge: 1}}
 	RolloutStrategy *RolloutStrategy `json:"rolloutStrategy,omitempty"`
+
+	// The RemediationStrategy that controls how control plane machines remediation happens.
```

Comment:
```suggestion
	// The RemediationStrategy that controls how control plane machine remediation happens.
```
(same below on the struct)